Scripts/sh/runTestsYAML.sh

```
@@ -0,0 +1,9 @@
+cd ../../
```

Shellcheck

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)